src/components/Form/CheckboxInput.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import T from 'prop-types'
+import { classNames } from '../../utils/'
+
+const CheckboxInput = ({ className, ...rest }) => (
+  <input className={classNames('c-form-input__input', className)} {...rest} />
+)
```

Comment:
We're missing some bits of the checkbox.  In nebula it's hidden away and always comes with the associated label to style it etc.
![screen shot 2017-08-08 at 13 27 46](https://user-images.githubusercontent.com/4982001/29071889-6a3db0d2-7c3d-11e7-9995-20dba75b610d.png)

HTML from Nebula Components:
```html
<div class="c-form-input">
  <input type="checkbox" class="c-form-input__input" id="check-1" value="on">
   <label for="check-1" class="c-form-input__label c-form-input__label--checkbox">
      Select
   </label>
</div>
```